src/main/java/edu/hm/hafner/analysis/parser/ClairParser.java

```diff
@@ -0,0 +1,119 @@
+package edu.hm.hafner.analysis.parser;
+
+import java.io.IOException;
+import java.io.Reader;
+import java.util.Locale;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.json.JSONTokener;
+
+import edu.hm.hafner.analysis.Issue;
+import edu.hm.hafner.analysis.IssueBuilder;
+import edu.hm.hafner.analysis.IssueParser;
+import edu.hm.hafner.analysis.ParsingException;
+import edu.hm.hafner.analysis.ReaderFactory;
+import edu.hm.hafner.analysis.Report;
+import edu.hm.hafner.analysis.Severity;
+
+/**
+ * A parser for dockerlint json output.
+ * <p>
+ * See <a href='https://github.com/arminc/clair-scanner'>clair-scanner</a> for project details.
+ *
+ * @author Andreas Mandel
+ */
+public class ClairParser extends IssueParser {
+    private static final long serialVersionUID = 42L;
+
+    String image;
+
+    @Override
+    public boolean accepts(final ReaderFactory readerFactory) {
+        return readerFactory.getFileName().endsWith(".json");
+    }
+
+    @Override
+    public Report parse(final ReaderFactory readerFactory) throws ParsingException {
+        Report report = new Report();
+        try (Reader reader = readerFactory.create()) {
+            JSONObject jsonReport = (JSONObject) new JSONTokener(reader).nextValue();
+            image = optStringIgnoreCase(jsonReport,"image");
```

Comment:
Better as local variable that is an additional parameter to `convertToIssue`

Reply:
I've just committed the change.